### PR TITLE
Locate entity filters with the transformers

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

This will co-locate the filters and transformers for each entity to hopefully ease this process and avoid merge conflicts.  It will also _automagically_ import them.

**TODO:**
Seeing as how the entity files are exporting a transformer function along with the array of items to be filtered on, I'd like to rename the `transformers` directory to something more appropriate.  I'll do some thinking on this.

## Testing done
- `npm run test:unit ./src/site/stages/build/process-cms-exports/tests/`

## Screenshots
N/A

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by Clark-W-Griswold — APPROVED** [review]
👍

**Review by cvalarida — APPROVED** [review]
I dig it. If you could also update [the generator template](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/process-cms-exports/generator-cms-content-model/templates/transformer#L12), that'd be awesome. (Also, we won't need [this bit](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/process-cms-exports/generator-cms-content-model/generators/app/index.js#L250-L255) anymore.)

File: src/site/stages/build/process-cms-exports/helpers.js [hunk 1]
```diff
@@ -42,6 +43,26 @@ module.exports = {
...
 
+  /**
+   * Dynamically read in exported properties
+   * and put them into an object indexed by filename
+   *
+   * @param {String} dir - The directory to import all files from
+   * @param {String} prop - The name of the exported property to put into the dict
+   * @return {Object} - The dict of filenam -> exported prop mappings
+   */
+  importPropToDict(dir, prop) {
```
[the remark this answers is not in the document]
Comment (bkjohnson) [reply src/site/stages/build/process-cms-exports/helpers.js]: That looks good.  When I was naming this I was having some trouble picking a good one :upside_down_face:

File: src/site/stages/build/process-cms-exports/generator-cms-content-model/templates/transformer [hunk 2]
```diff
@@ -9,4 +9,7 @@ const transform = entity => ({
     <%_ }); _%>
   },
 });
-module.exports = transform;
+module.exports = {
+  filters: [<%- rawPropertyNames.map(key => `'${key}'`).join(', ') %>],
```
Comment (cvalarida) [inline src/site/stages/build/process-cms-exports/generator-cms-content-model/templates/transformer]: :tada:

**Review by cvalarida — APPROVED** [review]

**Pull request merged**